src/shared/src/components/MainContentNav.tsx

```diff
@@ -0,0 +1,65 @@
+import { makeStyles } from "@material-ui/core";
+import React from "react";
+import cn from 'classnames';
+
+export interface IMainContentNavProps {
+  navigateText: string;
+}
+
+const useStyles = makeStyles({
+  button: {
+    textDecoration: 'underline',
+    textAlign: 'left',
+    width: 'fit-content',
+  },
+  jumpNav: {
+    backgroundColor: "#fff",
+  },
+  'sr-only': {
+    position: 'absolute',
+    width: '1px',
+    height: '1px',
+    padding: 0,
+    margin: '-1px',
+    overflow: 'hidden',
+    clip: 'rect(0,0,0,0)',
+    border: 0,
+  },
+  'sr-only-focusable': {
+    '&:focus': {
+      position: 'relative',
+      width: 'auto',
+      height: 'auto',
+      clip: 'initial',
+      margin: 'inherit',
+      padding: 'inherit',
+      border: 'initial',
+    },
+  }
+});
+
+export function MainContentNav({
+  navigateText,
+}: IMainContentNavProps) {
+  const classes = useStyles();
+
+  const handleOnClick = () => {
+    // workaround because we still use a hash-router (sigh...)
+    // can be replaced by the more elegant solution <a href="#main-content></a> once this is no longer the case.
+    const main = document.getElementById('main-content');
+    main.tabIndex = -1;
+    main.focus();
+  }
+
+  return (
+    <nav className={cn(classes.jumpNav)}>
```

Comment:
How will this `<nav>` scale to allow more items?
If this is the "Jump to landmark link list component", maybe a different name?
MainContentNav sounds like something that is displayed over the main content to me.
Maybe LandmarkShortcuts

Reply:
Great suggestion, renaming and refactoring to make scaling more straight forward.